chains/evm/contracts/ccvs/components/BaseOnRamp.sol

```diff
@@ -94,11 +94,13 @@ abstract contract BaseOnRamp is ICCVOnRampV1, ITypeAndVersion {
     }
   }
 
-  function _assertSenderIsAllowed(uint64 destChainSelector, address sender) internal view {
+  function _assertSenderIsAllowed(uint64 destChainSelector, address sender, address rampOnRouter) internal view {
```

Comment:
It's not rampOnRouter right? You want to check here IF it is the ramp on the router

Reply:
https://github.com/smartcontractkit/chainlink-ccip/pull/1182/commits/826eb502e9d11aebd9b9ad9a273b6af3813bae3e

Reply:
How about messageSender and verifierCaller?